src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -60,8 +62,12 @@ private void validateSnapshotDataNotEmpty(List<Map<String, Object>> values) {
         }
     }
 
+    private String uuidToIndexName(UUID id) {
```

Comment:
maybe factor out into util class? Seems generally useful

Reply:
I think it's unlikely that any non-search features will use this method.

Reply:
At some point we may have a few different things that map to index names. Also we had talked about storing the index name in snapshot in postgres. We can move it around later if it makes sense to.